src/datadog.rs

```
@@ -73,8 +73,9 @@ impl DatadogBuilder {
             self.global_tags.push(',');
         }
         let formatted_tag = format!("{key}:{value}");
-        let formatted_tag = serde_json::to_string(&formatted_tag).unwrap();
-        self.global_tags.push_str(&formatted_tag);
+        if let Ok(formatted_tag) = serde_json::to_string(&formatted_tag) {
+            self.global_tags.push_str(&formatted_tag);
```

if this fails, we would generate an invalid payload, in which case we might as well fail hard.

---

Pushed a fix for this, although this is also pretty much unreachable.